Sources/Components/BottomSheet/Controller/BottomSheet.swift

```diff
@@ -6,19 +6,23 @@ import UIKit
 
 extension BottomSheet {
     public struct Height {
-        public let compact: CGFloat
+        public let compact: CGFloat?
```

Comment:
Not sure if it's actually better, but I think we could achieve the same by making `compact` equal to `expanded`. Then you can use something like `isExpandedByDefault: Bool { return compact == expanded }` when needed. What do you think?

Reply:
Good idea! It would be nice to avoid the optional!